frontend/src/components/smart/PvPArenaSummary.vue

```diff
@@ -135,7 +145,10 @@ export default {
   data() {
     return {
       tab: 0,
-      dayjs
+      dayjs,
+      loading: true,
```

Comment:
By default loadings often are initialized to `false`, then on the action start, switched to `true`, after the action (often in `finally` block for actions that may fail) switched back to `false` again